genesis-template.json

```diff
@@ -11,13 +11,31 @@
     "petersburgBlock": 0,
     "istanbulBlock": 0,
     "muirGlacierBlock": 0,
+    "berlinBlock": 13996000,
+    "londonBlock": 22640000,
     "bor": {
-      "period": 1,
-      "producerDelay": 4,
+      "jaipurBlock": 22770000,
+      "period": {
+        "0": 1
+      },
+      "producerDelay": 6,
       "sprint": 64,
-      "backupMultiplier": 1,
+      "backupMultiplier": {
+        "0": 1
+      },
       "validatorContract": "0x0000000000000000000000000000000000001000",
-      "stateReceiverContract": "0x0000000000000000000000000000000000001001"
+      "stateReceiverContract": "0x0000000000000000000000000000000000001001",
+      "burntContract": {
+        "22640000": "0x70bcA57F4579f58670aB2d18Ef16e02C17553C38"
+      },
+      "blockAlloc": {
+        "22244000": {
```

Comment:
You don't need this because whenever we will create genesis and it will have the new contract from block 0.